polaris-core/src/main/java/io/polaris/core/PolarisConfiguration.java

```diff
@@ -111,7 +113,7 @@ public static <T> Builder<T> builder() {
       PolarisConfiguration.<Boolean>builder()
           .key("ALLOW_NAMESPACE_LOCATION_OVERLAP")
           .description(
-              "If set to true, allow one table's location to reside within another table's location. "
+              "If set to true, allow one namespace's location to reside within another namespace's location. "
```

Comment:
Unrelated change, but a good fix. Once we have more committers we should separate this sort of thing out into nit PRs just in case :)